translations/en-us.yml

```diff
@@ -689,6 +689,16 @@ list:
 contributor_list:
     x_more: '{x} more'
     anonymous: 'Anonymous contributors'
+    more: 'Load more contributors'
+    remove_contributor:
```

Comment:
Have we determined, going forward, if we want to use camelCase or snake_case for these?

Reply:
I can't recall a case being agreed upon for translations. My personal preference is camelCase 🐫

Reply:
We have not. Here, I just went with `snake_case` because the top-level key `contributor_list` was. I am in favor of `snake_case` for translation keys.
`kebab-case` for file names
`snake_case` for translation keys
`camelCase ` for variable names